src/Compilers/VisualBasic/Test/Syntax/Scanner/ScannerTests.vb

```diff
@@ -508,9 +508,39 @@ Public Class ScannerTests
         Assert.Equal(" _ ", tk.ToFullString())
 
         tk = ScanOnce(" _'", startStatement:=True)
-        Assert.Equal(SyntaxKind.BadToken, tk.Kind)
+        Assert.Equal(SyntaxKind.EndOfFileToken, tk.Kind)
         Assert.Equal(" _'", tk.ToFullString())
-        Assert.Equal(30999, tk.Errors.First().Code)
+        Assert.True(tk.LeadingTrivia(0).Kind = SyntaxKind.WhitespaceTrivia)
+        Assert.True(tk.LeadingTrivia(1).Kind = SyntaxKind.LineContinuationTrivia)
+        Assert.True(tk.LeadingTrivia(2).Kind = SyntaxKind.CommentTrivia)
+        Assert.Equal(37306, tk.Errors.First().Code)
```

Comment:
>Assert.Equal(37306, tk.Errors.First().Code) [](start = 8, length = 43)

Is the error attached to specific trivia? Please assert that fact

Reply:
I added some and also for variable whitespace between _ and '

Reply:
> Is the error attached to specific trivia? Please assert that fact

It doesn't look like this comment had any response. 

---
In reply to: [217797461](https://github.com/dotnet/roslyn/pull/28327#discussion_r217797461) [](ancestors = 217797461)